src/Yield/YieldTabs.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect } from 'react';
+import { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { Grid } from '@material-ui/core';
+import YieldTables from './YieldTables';
+
+const YieldCharts = (props) => {
+  /*useEffect(() => {
+    console.log(data);
+  }, []);*/
+
+  const { data, activeTable, farmYears, affiliations } = props;
+  const [tableData, setTableData] = useState(data);
+
+  useEffect(() => {
+    setTableData(data);
+  }, [data]);
+
+  if (activeTable === 'Corn') {
```

Comment:
maybe build this logic into YieldTables and have activeTable and dataUnavailableText be props? it seems like there's a lot of duplicated code here. it could just be one instance of YieldTable